docker/udp-server/Dockerfile

```diff
@@ -1,13 +1,24 @@
-FROM golang:1
-MAINTAINER Valentin Kuznetsov vkuznet@gmail.com
+FROM golang:1.20 AS build
+LABEL maintainer="Nikodemas Tuckus <tuckus.nikodemas@cern.ch>"
+
+ARG TAG=v00.00.00
+ENV WDIR=/data \
+    CGO_ENABLED=0
+
+WORKDIR $WDIR
+
+RUN git clone https://github.com/dmwm/udp-collector && \
+    cd udp-collector && \
+    if [ "$TAG" = "v00.00.00" ]; then git checkout master; else git checkout tags/$TAG -b build; fi && \
+    make
```

Comment:
I suggest to double check that you indeed compile static executable. In your Dockerfile you setup CGO_ENABLED in ENV, I usually put it in front of make, e.g. `CGO_ENABLED=0 make`. Just check with `ldd` command that final executable indeed a static one.